src/router/router.ts

```diff
@@ -1,19 +1,16 @@
-import Vue from 'vue'
-import VueRouter, { Route } from 'vue-router'
+import { Route, createWebHistory, createRouter } from 'vue-router'
 import { routes } from './routes'
 import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
 
 /**
  * Configures and returns Vue Router.
  */
 export function getVueRouter () {
-  Vue.use(VueRouter)
-
-  const router = new VueRouter({
-    mode: 'history',
+  const router = createRouter({
```

Comment:
Some changes here to the router since it's been upgraded. Reference: https://router.vuejs.org/guide/migration/